README.md

```diff
@@ -5,33 +5,31 @@
+The `spinDanceMove` function has `positonNumber` parameter which is basically the number of iterations or repetitions the programs characters will do permutations or number of times the programs will dance. As the aim of this method is to move the programs character from end to front so we have to first divide the programs standing in the line (like s2 on `abcde` produces `deabc`, here `abcde` should be divided into `abc` and `de`.) Now, we have to find the index from which the spin dance move starts (in `sX` on `abcde`, we have to find X) and for that we subtracted the `programsChar` list size from the `positionNumber`. To slice or cut the `programsChar` into two parts I used the slice property of the `mutableList` returns a list containing elements at indices in the specified indices range.
 
-We have to update the ```programsChar``` so we added both sliced part in such a way that the ending sliced part added in the beginning of the ```programsChar``` while we added the starting sliced part to the ending of the ```programsChar```. 
-
-         
-         fun spinDanceMove(positionNumber: Int) {
-                      programsChar = ((programsChar.slice((programsChar.size - positionNumber) until programsChar.size)) 
-                                                          + 
-                      programsChar.slice(0 until (programsChar.size positionNumber)))).toMutableList()
-                                }
-
-The ```exchangeDanceMove``` method had two parameters ```firstPosition``` and ```secondPosition ```parameters with Int datatypes. We will be required these two positions for ```programChar``` as the objective of creating this function is to simply exchange two ```programsChar``` at positions. To achieve this I first created a temp variable and stored the ```programsChar``` at first position. Then, in the next line I assigned the value of second position char to first position char and in third line we got real first position’s value in second position from that temp variable. 
+We have to update the `programsChar` so we added both sliced part in such a way that the ending sliced part added in the beginning of the `programsChar` while we added the starting sliced part to the ending of the `programsChar`. 
+```kotlin
+fun spinDanceMove(positionNumber: Int) {
+    programsChar = ((programsChar.slice((programsChar.size - positionNumber) until programsChar.size)) 
+    + programsChar.slice(0 until (programsChar.size positionNumber)))).toMutableList()
+}
+```
```

Comment:
Notice that I've wrapped this block of code in triple backticks and have given it the `kotlin` annotation (after the opening backticks on line 27). This provides you with proper syntax highlighting and makes your code more resadable.